ui/browsing/browsingpane.go

```diff
@@ -175,6 +179,7 @@ func (b *BrowsingPane) doSetPage(p Page) bool {
 	}
 	b.pageContainer.Remove(b.curPage)
 	b.pageContainer.Objects[1] = p
+	b.updateNavBtnsColor(p)
```

Comment:
This should be called in `SetPage` as well, in the special nil case (called when logging out from a server). Make sure to handle the nil case in updateNavBtnsColor so that if Page is nil it should set all buttons to foreground (default) color

Reply:
Thanks for the suggestions! Made those changes too